learnify/mobile/lib/features/auth/verification/view-model/verification_view_model.dart

```diff
@@ -65,6 +77,24 @@ class VerificationViewModel extends BaseViewModel {
   }
 
   Future<String?> _verificationRequest() async {
+    final String verificationCode = _verificationCodeFirstDigitController.text +
+        _verificationCodeSecondDigitController.text +
+        _verificationCodeThirdDigitController.text +
+        _verificationCodeFourthDigitController.text;
+
+    final VerifyEmailRequest requestModel =
+        VerifyEmailRequest(email: email, code: verificationCode);
+
+    final IResponseModel<VerifyEmailResponse> resp =
+        await _authService.verifyEmail(requestModel);
+
+    if (resp.hasError) return resp.error?.errorMessage;
+    await navigationManager.navigateToPage(
```

Comment:
Please convert this to navigateToPageAndClear() since we should clear all old routes (including verify email screen) after the authentication process.